t/id-keyword-draft6.t

```diff
@@ -0,0 +1,45 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+use JSON::Validator;
+
+my ($base_url, $jv, $t, @e);
```

Comment:
I think it's better if you declare this variables where you first assign to them. I also don't think all of them are in use?

Reply:
I copied this file from t/id-keyword-draft7.t thinking that this was the style you preferred. Shall I fix that file as well?

Reply:
That would be awesome. My style when declaring variables used to be quite consistent, but the last year, I've changed my style, so it might be a mess multiple places in the code 😞

Reply:
ok, I pushed a commit to tidy up t/id-keyword-draft7.t.